src/corona_hakab_model/consts.py

```diff
@@ -112,9 +112,15 @@ def from_file(param_path):
             data = read_file.read()
 
         parameters = eval(data, {"__builtins__": None, "dist": dist, "rv_discrete": rv_discrete})
+        Consts.sanitize_parameters(parameters)
 
         return Consts(**parameters)
 
+    @staticmethod
+    def sanitize_parameters(parameters):
+        for key, value in parameters.items():
+            assert isinstance(value, Hashable), f"{key} of value {value} in parameter file is unhashable"
```

Comment:
a user could enter as a value the tuple (12,{}), which is an instance of Hashable, but is still not hashable

The best solution is to try to hash all of `parameters` (no need to go element-by element) and fail if the hashing fails

Reply:
also, assert should only be used for sanity checking, not data validation

Reply:
Fixed - now throws InvalidParameterException